ui/src/app/cohort-search/search-group-list/search-group-list.component.tsx

```diff
@@ -126,7 +126,8 @@ function mapMenuItem(item: CriteriaMenu) {
     id,
     name,
     sortOrder,
-    standard: domainId === Domain.VISIT.toString() ? true : null,
+    // We can remove this check once the standardSource flag is set to true
+    standard: domainId === Domain.VISIT.toString() ? true : standard,
```

Comment:
We had this condition when the menu response did not include the `standard` flag, so I think we can go ahead and remove it and just pass the `standard` from the response.

Reply:
So in case the flag is off, will we still have the standard flag in menu response?

Reply:
`standard` is in the response with the flag off, but I'm not sure how valid the values are. Looks like they're all `false` currently. Maybe we just leave the condition for now and can remove it later if not needed.

Reply:
My main issue is if it is set to null instead of TRUE in case of enablestandardSource= false, will it cause an issue?

Reply:
I do not see any difference if i assign the standard flag so updating the code